add activities (#3)
OceanSprint/oceansprint.org: docs/activities.md

# Sprint Activities List

## Most popular

### Wine tasting - 35€

We get picked up by bus and we dive deeper into the unique style of how wine is produced on Lanzarote.

Choose four of your favourite styles from a portfolio of wines.

A bite of artisanal cheese and charcuterie will accompany your personalized tasting. The experience takes about 60 minutes.

### Vulcano hike - free

A classic of Ocean Sprint. We leave the venue on foot for a 1.5h hike to the nearest vulcano with great views over the nearby tows and coastline.

Beginner-friendly.

### Intro to surfing - 30€

Pickup by vans at the sprint venue and leave for Famara, the surfing mecca of Lanzarote. Two hours in the water with rented surfboard, wetsuits and surf coaching. Back by dinner.

Beginner-friendly.

### Bonfire Night in the Desert - free

A night under the stars with a bonfire in the middle of a vulcano. 

> We'll organize the transportation and each participants brings their own food&drinks.

Author: ```suggestion
We'll go for dinner near the venue, then organize the transportation and each participants brings their own drinks&snacks.
```

## Recommended

> ### Sailing - TBA

Author: ```suggestion
### Sailing with catamaran - 50€
```

> Enjoy the calm sea and scenic views during a few hour sailing excursion at sunset

Author: ```suggestion
Enjoy the ocean vibes and scenic views during a few hour catamaran excursion at sunset
```

# Sprint Activities List

## Most popular

### Wine tasting - 35€

> Dive deeper into the unique style of how wine is produced on Lanzarote.

Author: ```suggestion
We get picked up by bus and we dive deeper into the unique style of how wine is produced on Lanzarote.
```

Choose four of your favourite styles from a portfolio of wines.

> A bite of artisanal cheese and charcuterie will accompany your personalized tasting. The experience takes about 60 minutes.

Reviewer: Does this happen on-site, or a bus trip?

Author: bus trip to stravts

### Vulcano hike - free

> A classic of Ocean Sprint. We leave the venue for a 1.5h hike to the nearest vulcano with great views over the nearby tows and coastline.

Reviewer: ```suggestion
A classic of Ocean Sprint. We leave the venue on foot for a 1.5h hike to the nearest vulcano with great views over the nearby tows and coastline.
```

Beginner-friendly.

### Intro to surfing - 30€

Pickup by vans at the sprint venue and leave for Famara, the surfing mecca of Lanzarote. Two hours in the water with rented surfboard, wetsuits and surf coaching. Back by dinner.

Beginner-friendly.

### Bonfire Night in the Desert - free

A night under the stars with a bonfire in the middle of a vulcano. 

> We'll organize the transportation and each participants brings their own food&drinks.

Reviewer: How does "each participants brings their own food&drinks" work? They go to dino before we leave? 

Is this instead of dinner one day? What do the rest do?

Author: yeah, we'd get folks to dino.

I'd recommend we have dinner beforehand somewhere because it's annoying to eat there with 30 people and no tables.

Reviewer: Write that in the description. Dinner first, then off to dino for extras and drinks.

## Recommended

### Sailing - TBA

> Enjoy the calm sea and scenic views during a few hour sailing excursion at sunset

Reviewer: Calm sea lol :D

Author: chatgpt, wtf are you doing?

with drinks and tapas.

> ### Swimming - free

Reviewer: ```suggestion
### The Morning Dip - free
```

> Go for a morning swim 5min away from the venue.

Reviewer: Meet at venue at 8AM, walk down to the beach for a quick swim, be back by 9AM breakfast time.

> Go for a morning swim 5min away from the venue.

Reviewer: ```suggestion
Meet at venue at 8AM, walk down to the beach for a quick swim, be back by 9AM breakfast time.
```

### Yoga - free

> Depending on the interest, a yoga teacher could come to the venue or we would to go

Reviewer: ```suggestion
Depending on the interest, a yoga teacher could come to the venue or we would go to
```

### Diving - ~100€

On October 7, 1993, Lanzarote was granted the prestigious status of a Biosphere Reserve by UNESCO, making it the second Biosphere Reserve in the Canary Islands.

We invite you to embark on a journey of exploration and discovery as we delve into the remarkable underwater world of this biosphere. Lanzarote’s dive sites offer a unique opportunity to witness the diverse marine ecosystems that thrive in this protected area. From vibrant coral reefs to fascinating volcanic formations, each dive site holds its own charm and surprises.

Together, we can uncover the hidden treasures beneath the surface, encountering an abundance of marine life and experiencing the awe-inspiring beauty of Lanzarote’s biosphere. Join us in exploring these remarkable dive sites and witness firsthand the wonders that make this island a designated UNESCO Biosphere Reserve.

We can organize try dives for those who have not dived before as well as guided dives for already certified divers. For those coming early and/or staying longer it is also an option to make dive certificates.

### Bouldering - 12€

> Go to [a bouldering gym](https://hangonlanzarote.com/) and stretch those fingers.

Reviewer: Transport organized?

> Go to [a bouldering gym](https://hangonlanzarote.com/) and stretch those fingers.

Author: ```suggestion
We get picked up by vans and go to [a bouldering gym](https://hangonlanzarote.com/) and stretch those fingers.
```

### Submarine Excursion - 56€

> [A proper submarine experience](https://www.submarinesafaris.com/) that goes down to 30 meters, a geeky adventure underwater.

Reviewer: ```suggestion
[A proper submarine experience](https://www.submarinesafaris.com/) that goes down to 30 meters, a geeky adventure underwater.

Bus transport is provided by the submarine company.
```

### eMTB trip - 50€

> A 3-hour electric mountain-bike trip to the top of Lanzarote, with amazing 360 views followed by a 400m singletrack descent back to the sprint venue. @zupo's favorite after-work trail. Bikes rented from a nearby shop.

Reviewer: ```suggestion
A 3-hour electric mountain-bike trip to the top of Lanzarote, with amazing 360-degrees views followed by a 400 vertical-meters singletrack descent back to the sprint venue. @zupo's favorite after-work trail. Bikes rented from a nearby shop.
```

Experience needed.

### e-bike trip - 40€

> A 3 hour e-bike tour along the coastal pathway from Costa Teguise, through Arrecife the capital, past the airport to Puerto del Carmen, and back.

Reviewer: ```suggestion
A 3 hour e-bike tour along the coastal pathway from Costa Teguise, through Arrecife, the capital of Lanzarote, past the airport to Puerto del Carmen, and back.
```

Beginner-friendly.

> ### Intro to Wing-foiling / Windsurfing - 70€

Author: ```suggestion
### Intro to Wing-foiling / Windsurfing - 50€
```

> Walk down to the beach and do a 3-hour intro lesson to the new sport of Wing-foiling at the local windsurfing school.

Reviewer: ```suggestion
Walk down to the beach and do a 3-hour intro lesson to the new sport of Wing-foiling at the local windsurfing school.

Beginner friendly.
```

### Dawn patrol - free

> Beat the crowds and experience true Lanzarote surfing by joining @zupo/@domenkozar at locals-only spots around the island, while watching the Sun rise over the horizon.

Reviewer: ```suggestion
Beat the crowds and experience pure Lanzarote surfing by joining @zupo/@domenkozar at locals-only spots around the island, while watching the Sun rise over the horizon.
```
